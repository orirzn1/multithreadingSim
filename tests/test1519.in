L11
S6
O13
N6

T0
I@0x00000000
SUB $0, $5, $1
SUB $3, $6, $3
LOAD $1, $4, $5
SUBI $1, $2, 0x1
STORE $7, $4, $1
ADDI $7, $4, 0x4
ADDI $7, $7, 0x4
SUB $1, $7, $1
SUB $3, $7, $7
SUB $7, $5, $3
HALT $0

T1
I@0x00000000
SUBI $1, $4, 0x2
SUBI $1, $3, 0x2
ADDI $2, $6, 0x3
LOAD $2, $5, 0x3
HALT $0

T2
I@0x00000000
SUB $1, $2, $0
SUBI $1, $3, 0x3
SUBI $2, $6, 0x2
SUBI $3, $3, 0x2
HALT $0

T3
I@0x00000000
SUB $1, $5, $3
LOAD $1, $4, 0x2
ADD $1, $3, $4
SUB $0, $1, $2
ADD $3, $5, $6
SUBI $5, $1, 0x3
ADD $7, $5, $6
SUBI $6, $4, 0x3
HALT $0

T4
I@0x00000000
LOAD $0, $7, 0x3
STORE $4, $2, $5
ADD $0, $1, $0
SUB $7, $7, $2
SUBI $5, $4, 0x3
LOAD $2, $5, $2
SUBI $4, $2, 0x1
HALT $0

T5
I@0x00000000
LOAD $4, $7, $1
ADDI $5, $2, 0x4
LOAD $5, $6, $3
STORE $3, $6, $1
ADDI $0, $0, 0x1
LOAD $1, $2, 0x0
STORE $6, $4, $6
SUBI $2, $5, 0x2
ADDI $0, $2, 0x4
HALT $0

D@0x0
0x8
0x9
0x7
0x4
0x9
0x6
0x5
0x2
0x8
