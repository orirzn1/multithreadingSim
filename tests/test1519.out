
---- Blocked MT Simulation ----

Register file thread id 0:
	R0 = 0x0	R1 = 0x9	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 1:
	R0 = 0x0	R1 = 0xFFFFFFFE	R2 = 0x8	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 2:
	R0 = 0x0	R1 = 0xFFFFFFFD	R2 = 0xFFFFFFFE	R3 = 0xFFFFFFFE	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 3:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0xFFFFFFFD	R6 = 0xFFFFFFFD	R7 = 0xFFFFFFFD
Register file thread id 4:
	R0 = 0x8	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0xFFFFFFFF	R5 = 0xFFFFFFFD	R6 = 0x0	R7 = 0x0
Register file thread id 5:
	R0 = 0x2	R1 = 0x0	R2 = 0xFFFFFFFE	R3 = 0x0	R4 = 0x8	R5 = 0x0	R6 = 0x0	R7 = 0x0
Blocked MT CPI for this program 5.416667

-----Finegrained MT Simulation -----

Register file thread id 0:
	R0 = 0x0	R1 = 0x9	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 1:
	R0 = 0x0	R1 = 0xFFFFFFFE	R2 = 0x8	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 2:
	R0 = 0x0	R1 = 0xFFFFFFFD	R2 = 0xFFFFFFFE	R3 = 0xFFFFFFFE	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 3:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0xFFFFFFFD	R6 = 0xFFFFFFFD	R7 = 0xFFFFFFFD
Register file thread id 4:
	R0 = 0x8	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0xFFFFFFFF	R5 = 0xFFFFFFFD	R6 = 0x0	R7 = 0x0
Register file thread id 5:
	R0 = 0x2	R1 = 0x0	R2 = 0xFFFFFFFE	R3 = 0x0	R4 = 0x8	R5 = 0x0	R6 = 0x0	R7 = 0x0
Finegrained Multithreading CPI for this program 1.312500

